definitions.yaml

```diff
@@ -504,9 +504,6 @@ definitions:
       owner:
         type: string
         description: Name of the organization owning the cluster
-      release_version:
-        type: string
-        description: The semantic version number of this cluster
```

Comment:
@oponder I just noticed that this PR has removed The `release_version` attribute from the V4ClusterListItem model. That doesn't seem right. Could you please provide a follow-up to fix that?

Reply:
Oops, thats some mistake I must've done while merging. Fixing.